### ISSUE TITLE
Global Iconography changes

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/va.gov-team#75709]
_Update each sprint until completed_
| Date | Status | Launch Date (see above) | Notes |
| ----- | ------ | ------------------------- | ------ |
| 17 Oct | in-progress | on-track | one remaining PR |

## Description
[Original RFC](https://vfs.atlassian.net/wiki/spaces/DST/pages/2775253429/RFC+Migration+Strategy+Formation+to+CSS-Library)

This initiative tracks the progress of sunsetting Formation in favor of the new CSS Library which will house all of the styles (element selectors, utility classes, variables, tokens, etc) VA.gov imports and uses. There are 4 high level steps in this migration:
1. Set up [CSS Library](https://github.com/department-of-veterans-affairs/component-library/tree/main/packages/css-library) so it's maintained separately from the component library.
2. Migrate global base styles from Formation to CSS Library
3. Migrate application utility class usage 
4. Archive Formation

High Level User Story/ies
As a VFS team member, I want the VA design system iconography to be up-to-date so that my apps are using the USWDS icons. 

Hypothesis or Bet
If we update the icons globally, all apps will be using the USWDS inconography so all VFS team apps are using the USWDS icons so there is consistency across VA.gov. 

OKR
Quarterly Operations and Maintenance Delivery - (Q1) FY Q2 2024

Definition of done
What must be true in order for you to consider this epic complete?
We have updated the VA DS to USWDS icons.

**Comment by humancompanion-usds** [comment]
The goal is to make va-icon available to all teams by next Wednesday 5/1. We're having some environment/build difficulties that are delaying this. Once that component is working will officially ask teams to start using it and migrate to the USWDS icon set. The deadline for switching will be 6/19, which will officially mark the day we can declare victory on upgrading to USWDS v3!!!!

**Comment by humancompanion-usds** [comment]
## Iconography: Start moving to USWDS icon set as of Wednesday May 1st, deadline to shift Wednesday June 19th

* The ask: Switch to using the va-icon web-component, or if you cannot use a web-component please shift to using the appropriate SVG. 
* The new icon set is available in [Figma](https://www.figma.com/file/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?type=design&node-id=293%3A6211&mode=design&t=unKDZLSetqWFsv0R-1) and in [guidance](https://design.va.gov/foundation/icons).
* v3 web-components now are using the USWDS icons in production (as of 4/26). The rest of the Design System web-components will be updated on May 22nd.
* FontAwesome will be removed on June 19th.

**Comment by caw310** [comment]
Q2 Update
- va-icon and USWDS icon set available for use as of May 1st. 
- FontAwesome will be removed June 26.

**Comment by humancompanion-usds** [comment]
@caw310 and @micahchiang - Let's discuss next Tuesday 9/10 whether we can close this initiative out.

**Comment by humancompanion-usds** [comment]
* 1 PR waiting on CI that once it merges will account for the remaining instances (1) in Financial status report.
* Another PR accounts for the other instance in Facility Locator. 

Once those 2 merge we'll have no more FontAwesome in vets-website!

Proxy rewrite application doesn't use icons but references a FA string path in Formation, which isn't being used, but needs to be removed. That's causing the PR not to build.

**Comment by humancompanion-usds** [comment]
@caw310 and @micahchiang - Can we close this?

**Comment by caw310** [comment]
@humancompanion-usds There is still one outstanding PR that needs to be addressed. Micah will try to look into that this week so we can close this issue.

**Comment by humancompanion-usds** [comment]
All actual icons have been removed from vets-website and content-build. 

There is one remaining PR that cleans up import paths that needs to be merged and then this can be closed.

**Comment by humancompanion-usds** [comment]
Agreed to get this done in the next sprint.

**Comment by humancompanion-usds** [comment]
@micahchiang and @rmessina1010 - Are we done with this? If not, can you detail what remains? Thanks!

**Comment by caw310** [comment]
Work for this epic is done.
